caffe2/onnx/helper.cc

```diff
@@ -5,8 +5,6 @@
 
 namespace caffe2 { namespace onnx  {
 
-size_t DummyName::counter_ = 0;
-
 std::unordered_set<std::string>& DummyName::get_used_names() {
   static std::unordered_set<std::string> used_names;
```

Comment:
We probably should also make this a class member variable?

Reply:
Crap. I forgot to change this!